### DISCUSSION TITLE
一直显示暂停下载

### DISCUSSION THREAD

**Discussion by Lapuris** [qiye45/wechatDownload#265] [Q&A]
![微信图片_20241221093650](https://github.com/user-attachments/assets/bc627c5a-9c7b-4ad1-b3ee-260bb7bd8bbb)
爬了六百多篇后就反复显示暂停下载，想问一下怎么解决呀

**Comment by qiye45** [comment]
https://github.com/qiye45/wechatDownload/issues/9#issuecomment-1889249416
